packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
+import React from 'react';
 import CategoryList from '../../components/CategoryList';
 
-export default class CMS extends Component {
-    render() {
-        return <CategoryList title="Shop by category" id={2} />;
-    }
-}
+const CMS = () => {
+    return <CategoryList title="Shop by category" id={2} />;
+};
+
+export default CMS;
```

Comment:
Some of these are pretty simple but also perhaps unnecessary. This is one of those examples. It may be preferable not to do _all_ components and instead focus on ones that use some context/connected logic or functions provided by a container. These will eventually move to hooks in the scope of the peregrine migration milestone.